src/swift/Sources/Models/TokenFile.swift

```diff
@@ -229,465 +234,430 @@ class TokenFile: Codable {
         keyword(value: "class")
         whitespace()
         type(name: decl.name.textDescription, definitionId: defId)
-        if let genericParam = decl.genericParameterClause {
-            handle(clause: genericParam)
-        }
-        if let inheritance = decl.typeInheritanceClause {
-            handle(clause: inheritance)
-        }
-        if let genericWhere = decl.genericWhereClause {
-            handle(clause: genericWhere)
-        }
+        handle(clause: decl.genericParameterClause)
+        handle(clause: decl.typeInheritanceClause)
+        handle(clause: decl.genericWhereClause)
+        whitespace()
         punctuation("{")
+        newLine()
         indent {
-            for member in decl.members {
-                switch member {
-                case .declaration(let decl):
-                    process(decl)
-                default:
-                    continue
-                }
+            decl.members.forEach { member in
+                handle(member: member)
             }
         }
-        if needsNewLine {
-            newLine()
-        }
         punctuation("}")
+        newLine()
+        return true
     }
 
-    private func process(_ decl: StructDeclaration) {
-        SharedLogger.debug("Struct Declaration")
-        SharedLogger.debug(decl.textDescription)
-        guard publicModifiers.contains(decl.accessLevelModifier ?? .internal) else {
-            return
-        }
-        let value = (decl.accessLevelModifier ?? .internal).textDescription
+    private func process(_ decl: StructDeclaration, overridingAccess: AccessLevelModifier? = nil) -> Bool {
```

Comment:
Ditto: Unsure about declaration changes

Reply:
I'm not sure what you mean about declaration changes.

Reply:
You answered my concerns above